### ISSUE TITLE
Expand "Strive for universal accessibility, be accessible by default"

### ISSUE THREAD

**Issue by equivalentideas** [equivalentideas/independent-archive-principles#7]
This would be a good place for a discussion of many of the different angles of accessibility. I think the intro par should be a bit more general and then the body can go through a list of considerations, including the ones that are in there currently: machine readability and legal access.
